api/_examples/alert-rules/main.go

```diff
@@ -39,6 +39,8 @@ func main() {
 		Severities:      api.AlertRuleSeverities{api.AlertRuleSeverityHigh},
 		ResourceGroups:  []string{"TECHALLY_000000000000AAAAAAAAAAAAAAAAAAAA"},
 		EventCategories: []string{"Compliance"},
```

Comment:
Should we have more tests of the newly added categories etc?

Reply:
updated alert_rule_test.go

Integration testing for new fields in the terraform-provider integrations too.